src/mon/MonCommands.h

```diff
@@ -301,18 +301,18 @@ COMMAND("mds versions",
 	"check running versions of MDSs",
 	"mds", "r", "cli,rest")
 COMMAND_WITH_FLAG("mds tell " \
-	"name=who,type=CephString " \
+	"name=role,type=CephString " \
 	"name=args,type=CephString,n=N", \
 	"send command to particular mds", "mds", "rw", "cli,rest", FLAG(OBSOLETE))
 COMMAND("mds compat show", "show mds compatibility settings", \
 	"mds", "r", "cli,rest")
-COMMAND_WITH_FLAG("mds stop name=who,type=CephString", "stop mds", \
+COMMAND_WITH_FLAG("mds stop name=role,type=CephString", "stop mds", \
 	"mds", "rw", "cli,rest", FLAG(DEPRECATED))
-COMMAND("mds deactivate name=who,type=CephString",
+COMMAND("mds deactivate name=role,type=CephString",
         "clean up specified MDS rank (use with `set max_mds` to shrink cluster)", \
 	"mds", "rw", "cli,rest")
 COMMAND_WITH_FLAG("mds set_max_mds " \
-	"name=maxmds,type=CephInt,range=0", \
+	"name=role,type=CephInt,range=0", \
```

Comment:
Accident?

Reply:
@batrick Reverted it, but it was not an accident. I misunderstood that [L1574](https://github.com/joscollin/ceph/blob/798b4b584a6635653065d430fe49bc2c0b97a5b4/src/mon/MDSMonitor.cc#L1574) in MDSMonitor.cc is for `mds set_max_mds`. But now I think L1574 in MDSMonitor.cc should be dropped, as `cmd_getval()` is called again inside the `if`. What do you suggest ?

Reply:
@batrick Alright. I re-based it and that entire function `int MDSMonitor::legacy_filesystem_command(` was removed in the ceph/master.